gnucash/import-export/aqb/gnc-gwen-gui.c

```diff
@@ -170,9 +170,16 @@ static gboolean keep_alive(GncGWENGui *gui);
 static void cm_close_handler(gpointer user_data);
 static void erase_password(gchar *password);
 static gchar *strip_html(gchar *text);
+#ifndef GWENHYWFAR5
```

Comment:
While the functions you're using do come from Gwen5, the actual chiptan functionality belongs to AQB6 so I think that's a more appropriate check; it in turn requires Gwen5 so if AQB6 is available so is Gwen5.

Reply:
Wow - thank you for the timely and elaborate comments :-) I have little experience with glib programming, so it was very helpful for me.
I will rework the code and then come back...